### PR TITLE
Update the help section #5698

### PR DESCRIPTION
**Pull request by aka001**

### CODE REVIEW

**Comment by aka001** [comment]
Is giving link to loomio fine under "What if my question is not answered in this FAQ? Where else can I get support?"

**Comment by Flaburgan** [comment]
No, we don't want support request on Loomio. You can redirect to the FAQs on the wiki though.

File: config/locales/diaspora/en.yml [hunk 1]
```diff
@@ -412,13 +413,14 @@ en:
...
       get_support_a_irc: "Join us on %{irc} (live chat)"
+      get_support_a_faq: "Read our %{faq} page on wiki"
       get_support_a_hashtag: "Ask in a public post on diaspora* using the %{question} hashtag"
     account_and_data_management:
       title: "Account and data management"
       move_pods_q: "How do I move my seed (account) from one pod to another?"
       move_pods_a: "In the future you will be able to export your seed from a pod and import it on another, but this is not currently possible. You could always open a new account and add your contacts to aspects on that new seed, and ask them to add your new seed to their aspects."
       download_data_q: "Can I download a copy of all of my data contained in my seed (account)?"
-      download_data_a: "Yes. At the bottom of the Account tab of your settings page there are two buttons for downloading your data."
+      download_data_a: "Yes. At the bottom of the Account tab of your settings page there is a button for downloading your data."
```
Comment (aka001) [inline config/locales/diaspora/en.yml]: I found only one button at end of account tab page "Request my profile data".

**Comment by aka001** [comment]
I have updated the pr. Kindly look at it.

File: config/locales/diaspora/en.yml (hunk 1 above)
Comment (jhass) [inline config/locales/diaspora/en.yml]: Mmmh, #5685 is going to readd the second one though.

Comment (aka001) [inline config/locales/diaspora/en.yml]: I have updated the pr.

**Comment by jhass** [comment]
I don't think we live up to the promise of submitting messages with `Ctrl+enter` everywhere yet, but we should so that only adds more demand to add it :P

Thanks!

**Comment by aka001** [comment]
Got it. :P

**Comment by aka001** [comment]
@jhass Shouldn't we have a point in help section about "Getting started as a developer" or "Want to contribute to diaspora?" to help new developers?

**Comment by jhass** [comment]
I don't think so, the help section is user oriented, not developer oriented.

As for login & password, that would first need us to think about how the user can find it while not being logged in.

**Comment by aka001** [comment]
IMHO, The help section must be visible to every user(irrespective of whether user is logged-in or not).

**Pull request merged**